magento/magento2
# Fix #21725 - Render price on category list with correct precision for current locale

### Description (*)

This PR changes price rendering on category view to include correct precision for current locale and currency.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#21725

### Manual testing scenarios (*)

1. Magento 2.3.0
2. Default locale set should be for Oman (ar_OM it's not by default available in Magento, need to be added in \Magento\Framework\Locale\Config but didn't check all locales to verify where it also occurs), Oman Rial (OMR) selected as currency
3. Visit product listing page - see that the products have prices with precision 3.
4. Visit product details page - see that the products have prices with precision 3.

### Contribution checklist (*)
